app/src/androidTest/java/com/example/mindyourpet/PetDatabaseTest.kt

```diff
@@ -0,0 +1,102 @@
+package com.example.mindyourpet
+
+import androidx.room.Room
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import com.example.mindyourpet.database.Pet
+import com.example.mindyourpet.database.PetDatabase
+import com.example.mindyourpet.database.PetDatabaseDao
+import com.example.mindyourpet.database.Reminder
+import kotlinx.coroutines.runBlocking
+import org.junit.Assert.assertEquals
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.io.IOException
+import kotlin.jvm.Throws
+
+
+/**
+ * This is not meant to be a full set of tests. For simplicity, most of your samples do not
```

Comment:
Was this comment written by you or copied from somewhere else? I don't think it belongs.

Reply:
I believe copied since Jennifer was in charge of this portion of the project

Reply:
I am not sure but I think I commented on another PR my reply.

Reply:
I am sorry I saw this already but I think I replied at a different place.